charts/shoot-core/components/charts/vpn-shoot/templates/psp/vpn-shoot-clusterrole.yaml

```
@@ -1,14 +1,8 @@
+# TODO(shafeeqes): Delete this Helm chart in a future version.
+---
 apiVersion: {{ include "rbacversion" . }}
 kind: ClusterRole
 metadata:
   name: gardener.cloud:psp:kube-system:vpn-shoot
-rules:
-- apiGroups:
-  - policy
-  - extensions
-  resourceNames:
-  - gardener.kube-system.vpn-shoot
-  resources:
-  - podsecuritypolicies
-  verbs:
-  - use
+  annotations:
+    resources.gardener.cloud/mode: Ignore
```

Can you leave 1 empty line at the end of the file?

```suggestion
    resources.gardener.cloud/mode: Ignore

```